<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小说下载工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,5">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>笔趣阁</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_biquge">
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>作者</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>最新章节</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下载</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>笔趣阁2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_biquge_2">
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>作者</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>最新章节</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下载</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>书趣阁</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_shuquge">
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>作者</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>最新章节</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下载</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>苗疆</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_miaojiang">
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>作者</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>最新章节</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>下载</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
